src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -434,5 +436,15 @@ private static async Task<string> AddBundleConfig(string hostJsonContent)
             hostJsonObj.Add("extensionBundle", bundleConfig);
             return JsonConvert.SerializeObject(hostJsonObj, Formatting.Indented);
         }
+
+
+        private static async Task<string> AddLoggingConfig(string hostJsonContent)
```

Comment:
any reason for this approach as opposed to just have it in the default host.json?

Reply:
I followed the sample of setting bundles as default in host.json. I think the only advantage we get by following this approach, is that we can introduce a flag to skip this logic in the init command `no-loggingConfig`. While that made sense for bundles, I don't think it does for Application Insight configuration. I will revert this change to simple add this configuration to default host.json

Reply:
sounds good